source/fundamentals/auth.txt

```diff
@@ -228,23 +228,34 @@ connect using ``MONGODB-CR``.
 ~~~~~~~~~~~~~~~
 
 .. note::
-   The MONGODB-AWS authentication mechanism is only available in MongoDB
-   versions 4.4 and later.
+
+   The MONGODB-AWS authentication mechanism is available in MongoDB
+   Atlas.
 
 The ``MONGODB-AWS`` authentication mechanism uses your Amazon Web Services
 Identity and Access Management (AWS IAM) credentials to authenticate your
 user.
 
+You can store your AWS credentials as environment variables, or insert
+them inline like the examples below. The driver checks for your credentials
+in the following order:
+
+1. Supplied values in a ``MongoCredential`` object.
```

Comment:
In practice, 1 and 2 are more an either/or type of thing.  If there is an order, it's under the application's control as to when they call MongoClientSettings.applyConnectionString.

Reply:
I've clarified this by combining them into a single item.